Casks/usr-sse2-rdm.rb

```diff
@@ -0,0 +1,16 @@
+cask "usr-sse2-rdm" do
+  version "2.5.0"
+  sha256 "9d583e43742018e45bad79c8af9e0accec6b6a10d94c60ac7d31721bbb3c237e"
+
+  url "https://github.com/usr-sse2/RDM/releases/download/#{version}/RDM.zip"
```

Comment:
Let's add appcast too :bulb: 
```suggestion
  url "https://github.com/usr-sse2/RDM/releases/download/#{version}/RDM.zip"
  appcast "https://github.com/usr-sse2/RDM/releases.atom"
```

Reply:
I don't think appcasts are needed when a livecheck is available?

Reply:
Hm, am I getting out of HB(C) contributing shape again? :smile:

Reply:
Doing cursory docs glance doesn't really reveal the answer to this question :thinking:

Reply:
See the `note` here: https://docs.brew.sh/Cask-Cookbook#stanza-appcast

Reply:
I did a Github search for appcast references in markdown files, OFC that part you just referenced got collapsed inside the search result. :facepalm: